src/routes/filfund/components/FundRingWidget/Connected.svelte

```diff
@@ -0,0 +1,171 @@
+<script lang="ts">
+  import { utils } from 'ethers'
+  import { FilsnapAdapter } from 'filsnap-adapter'
+  import { RPC } from 'iso-filecoin/rpc'
+  import { Token } from 'iso-filecoin/token'
+  import { onMount } from 'svelte'
+
+  import BrandLogoSmall from '$components/icons/BrandLogoSmall.svelte'
+  import { addNotification } from '$lib/notifications'
+  import { NETWORK_MAP } from '$lib/contract'
+
+  export let snap = null
+  export let title: string = 'Help fund us!'
+  export let description: string =
+    'If you rely upon Fund Ring’s efforts to keep your project going, please consider supporting our funding goal. Every little bit helps.'
+
+  let fetchingData = false
+  let loading = false
+  let totalFundsRaised = 0
+
+  const FUNDRING_CONTRACT_OWNER_ADDRESS =
+    't410ftgo7vbcywlxf72blsnr4tgmpvcxrs6icg34pouq'
+  const LOADING_TEXT = ['processing', 'sit tight', 'network speed may vary']
+  const RPC_ENDPOINT = 'https://api.calibration.node.glif.io/rpc/v1'
+
+  const rpc = new RPC({
+    api: RPC_ENDPOINT,
+    network: 'testnet'
+  })
```

Comment:
The new adapter version has a `rpc()` method that returns a rpc client configured by the filsnap.